core/src/main/scala/org/apache/spark/api/python/PythonUtils.scala

```diff
@@ -27,12 +27,15 @@ import org.apache.spark.SparkContext
 import org.apache.spark.api.java.{JavaRDD, JavaSparkContext}
 
 private[spark] object PythonUtils {
+  val PY4J_ZIP_NAME = "py4j-0.10.7-src.zip"
```

Comment:
this is better... is there a way not to hard code this at all? set from project properties?

Reply:
Not sure if that works in production. Maybe that could work somehow but we need to update that project's property anyway :).